db/migrate/20180801080615_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name,
+      t.string :email,
```

Comment:
@ivanraybuglosa When I run `rails db:migrate` it displays error, because `t.string` is a method, so no need to add a comma `,` to separate the attributes.